x-pack/legacy/plugins/reporting/export_types/csv/server/execute_job.test.js

```diff
@@ -359,11 +347,13 @@ describe('CSV Execute Job', function() {
         conflictedTypesFields: undefined,
         searchRequest: { index: null, body: null },
       };
-      await expectRejectedPromise(executeJob('job123', jobParams, cancellationToken));
+      await expect(
+        executeJob('job123', jobParams, cancellationToken)
+      ).rejects.toMatchInlineSnapshot(`[TypeError: Cannot read property 'indexOf' of undefined]`);
```

Comment:
this seems a little strange. It's worth a look to see if this is the actual expected error